test/data/marriage-abroad-responses-and-expected-results.yml

```diff
@@ -22931,7 +22931,7 @@
   - uk
   - partner_british
   - opposite_sex
-  :next_node: :outcome_opposite_sex_marriage_in_other_countries
+  :next_node: :outcome_opposite_sex_marriage_in_yemen
```

Comment:
Ah. That explains it. I had assumed if I expanded the diff on GitHub then I could see all the context, but I now see that even when I fully expand the diff, some lines are not included.